Admin/CategoryAdmin.php

```diff
@@ -33,6 +35,26 @@ public function configureRoutes(RouteCollection $routes)
     /**
      * {@inheritdoc}
      */
+    public function getFormBuilder()
+    {
+        // NEXT_MAJOR: set constraints unconditionnally
+        if (isset($this->formOptions['cascade_validation'])) {
+            unset($this->formOptions['cascade_validation']);
+            $this->formOptions['constraints'][] = new Valid();
+        } else {
+            @trigger_error(<<<EOT
+Unsetting cascade_validation is deprecated since 3.x, and will give an error in 4.0.
+Override getFormBuilder() and remove the "Valid" constraint instead.
+EOT
+            , E_USER_DEPRECATED);
+        }
```

Comment:
Yeah, probably not ideal indeed, but I can't think of another way either. Plus, I'm not sure many people would actually want to disable the validation. What do you think?